content/influxdb/v2.0/query-data/execute-queries/influx-api.md

```diff
@@ -1,54 +1,13 @@
 ---
-title: Execute queries
-seotitle: Different ways to query InfluxDB
-description: There are multiple ways to query data from InfluxDB including the InfluxDB UI, CLI, and API.
-weight: 103
+title: Query with the InfluxDB API
+weight: 202
 menu:
   influxdb_2_0:
-    name: Execute queries
-    parent: Query data
+    name: Query with the InfluxDB API
+    parent: Execute queries
 influxdb/v2.0/tags: [query]
 ---
 
```

Comment:
General comment for this doc – we may want to introduce the `/query` compatiblity endpoint in this doc if users want to query with InfluxQL.